src/Entity/IdeasWorkshop/Idea.php

```diff
@@ -639,6 +642,29 @@ public function decrementVotesCount(int $increment = 1): void
         $this->votesCount -= $increment;
     }
 
+    /**
+     * @SymfonySerializer\Groups({"idea_list_read", "idea_read"})
+     */
+    public function getNewCommentsCount(): int
```

Comment:
Why did you add this method that returns `$commentsCount` value as `getCommentsCount `?